.github/ISSUE_TEMPLATE/confiq.yml

```
@@ -0,0 +1,30 @@
+name: meta/repository Report
```

Can we add this emoji here?
```suggestion
name: 📇 Meta/Repository Feature
```

---

@errorline99
This one isn't resolved yet

---

finished it. thank you very much for explaining these stuff to me